hdt-lib/src/hdt/BasicHDT.cpp

```diff
@@ -777,7 +778,9 @@ size_t BasicHDT::loadMMapIndex(ProgressListener *listener) {
 
     // Get path
     string indexFile(fileName);
-    indexFile.append(".index");
+    indexFile.append(".index.v");
+		indexFile.append(HDT_VERSION);
+		indexFile.append(INDEX_VERSION);
```

Comment:
Not even [this one](http://www.cplusplus.com/reference/string/string/operator+/)?